src/views/home/topCities.jsx/TopCities.jsx

```diff
@@ -0,0 +1,30 @@
+import { FaEye } from "react-icons/fa";
+import { Link } from "react-router-dom";
+import { CityPart } from "./CityPart";
+
+
+export const TopCities = () => {
+    return (
+      <div className="container mt-40">
+        <div className="flex justify-center">
+          <h2 className="text-4xl font-bold">Top Cities by</h2>
+          <span className="text-dark-blue italic text-4xl font-bold ml-2">
+          Ads
+          </span>
+        </div>
```

Comment:
Since span not inherently represent anything, and it's used mostly for styling I think it should be inside h2 tag. Additionally, I think, but I'm not sure text-4xl class in span wouldn't be necessary then.

Reply:
+